webhook/main.go

```diff
@@ -34,28 +35,29 @@ var log = logf.Log.WithName("cmd")
 func main() {
 	logf.SetLogger(zap.Logger())
 
-	clusterConfig, err := rest.InClusterConfig()
+	// Get a config to talk to the apiserver
+	cfg, err := config.GetConfig()
 	if err != nil {
-		log.Error(err, "Failed when attempting to retrieve in cluster config")
+		log.Error(err, "")
```

Comment:
```suggestion
		log.Error(err, "Failed when retrieving config")
```

Reply:
I think `""` here is to be inlined with controller/main.go and the motivation behind it, I think - error should already have clear message when it happened...

But I assume there might be some errors thrown like `value must not be nil`, which won't show the place where it happened on controller level. So, I'm not +1 to duplicate message in the way Joshua suggests but it can be done as separate PR for all places where we log `""`

Reply:
Yeah woops, this was copied from the main controller code. I can't recall if it's autogenerated or not :smile: